docker-compose.yml

```diff
@@ -45,3 +47,20 @@ services:
             - apto-mariadb-volume:/var/lib/mysql
         networks:
             - apto-docker-network
+        ports:
+            - ${APTO_DOCKER_MARIADB_PORT}:3306
+    apto-pma-service:
+        networks:
+            - apto-docker-network
+        restart: always
+        container_name: apto-pma-container-${APTO_DOCKER_HOST}-${APTO_DOCKER_PORT}
+        hostname: localhost
```

Comment:
shouldn't the value be "${APTO_DOCKER_HOST}" here?

Reply:
no